plugins/source/azure/resources/services/cognitiveservices/resource_skus.go

```diff
@@ -0,0 +1,38 @@
+package cognitiveservices
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/cognitiveservices/armcognitiveservices"
+	"github.com/cloudquery/cloudquery/plugins/source/azure/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+)
+
+func ResourceSKUs() *schema.Table {
+	return &schema.Table{
+		Name:                 "azure_cognitiveservices_resource_skus",
+		Resolver:             fetchResourceSKUs,
+		PostResourceResolver: client.LowercaseIDResolver,
+		Description:          "https://learn.microsoft.com/en-us/rest/api/cognitiveservices/accountmanagement/resource-skus/list?tabs=HTTP#resourcesku",
+		Multiplex:            client.SubscriptionMultiplexRegisteredNamespace("azure_cognitiveservices_resource_skus", client.Namespacemicrosoft_cognitiveservices),
+		Transform:            transformers.TransformWithStruct(&armcognitiveservices.ResourceSKU{}),
```

Comment:
these don't have an ID. In `azure_compute_skus` we calculate a hash. We could do this here too.